dev/io.openliberty.security.fips.internal_fat/fat/src/io/openliberty/security/fips/fat/tests/fips1403/server/FIPS1403ServerTest.java

```diff
@@ -0,0 +1,198 @@
+            server.copyFileToLibertyServerRoot("publish/resources", "resources", STANDALONE_FIPS_PROFILE_FILENAME);
+            opts.put("-Dsemeru.fips", "true");
+            opts.put("-Dsemeru.customprofile", "OpenJCEPlusFIPS.FIPS140-3-Custom");
+            opts.put("-Djava.security.properties", server.getServerRoot() + "/resources/" + STANDALONE_FIPS_PROFILE_FILENAME);
+        // IBM SDK 8
+        } else {
+            isIBMJava8 = true;
+            opts.put("-Xenablefips140-3", null);
+            opts.put("-Dcom.ibm.jsse2.usefipsprovider", "true");
+            opts.put("-Dcom.ibm.jsse2.usefipsProviderName", "IBMJCEPlusFIPS");
+        }
+        server.setJvmOptions(opts);
+        server.startServer();
+        checkLogForFipsEnablementMessage(server, expectedProvider);
+    }
+
+    @Test
+    public void fips140_3EnvVarTest() throws Exception {
+        if(!GLOBAL_FIPS) {
+            server.copyFileToLibertyServerRoot("publish/resources", "resources" , LIBERTY_APPLICATION_FIPS_PROFILE_FILENAME);
+            server.addEnvVar(ENABLE_FIPS140_3_ENV_VAR, server.getServerRoot()+"/resources/" + LIBERTY_APPLICATION_FIPS_PROFILE_FILENAME);
+        }
+        server.startServer();
+        checkLogForFipsEnablementMessage(server, expectedProvider);
+    }
+
+    /**
+     * FIPS should still be enabled, but will use just the baseline Liberty profile when starting
+     * @throws Exception
+     */
+    @Test
+    public void fips140_3EmptyEnvVarTest() throws Exception {
+        assumeThat(GLOBAL_FIPS, is(false));
+        Path path = Paths.get(server.getServerRoot() + "/"+ SERVER_ENV_FILE);
+        String serverEnv = VAR_EXPANSION_ENV + System.lineSeparator() + ENABLE_FIPS140_3_ENV_VAR + "=\"\"";
+        Files.write(path, serverEnv.getBytes(StandardCharsets.UTF_8));
+        server.startServer();
+        checkLogForFipsEnablementMessage(server, expectedProvider);
+    }
+
+    @Test
+    public void fips140_3DirectoryQuotedTest() throws Exception {
+        assumeThat(GLOBAL_FIPS, is(false));
+        server.copyFileToLibertyServerRoot("publish/resources", "resources" , LIBERTY_APPLICATION_FIPS_PROFILE_FILENAME);
+        Path path = Paths.get(server.getServerRoot() + "/"+ SERVER_ENV_FILE);
+        String serverEnv = VAR_EXPANSION_ENV + System.lineSeparator() + ENABLE_FIPS140_3_ENV_VAR + "=\"" + server.getServerRoot() + "/resources/" + LIBERTY_APPLICATION_FIPS_PROFILE_FILENAME + "\"";
+        Files.write(path, serverEnv.getBytes(StandardCharsets.UTF_8));
+        server.startServer();
+        checkLogForFipsEnablementMessage(server, expectedProvider);
+    }
+
+    /**
+     * Test that a non-quoted path with a space in is correctly handled by Windows
+     *
+     * @throws Exception
+     */
+    @Test
+    public void fips140_3DirectorySpaceNoQuotesTest() throws Exception {
+        assumeThat(GLOBAL_FIPS, is(false));
+        assumeThat(server.getMachine().getOperatingSystem(), is(OperatingSystem.WINDOWS));
```

Comment:
i think we can take out this assumes windows line, since space with no quotes (and no variable expansion) should work on non-windows as well.